.github/workflows/pr-manual-merge.js

```diff
@@ -0,0 +1,79 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+async function main({ github, context } = {}) {
```

Comment:
This _should_ work, I'm not certain though. Would only assign the values from getLocalConfigs when github/context are otherwise undefined
```suggestion
async function main({ github = getLocalConfigs().github, context = getLocalConfigs().context } = {}) {
```